candy-machine/program/tests/core/master_edition_v2.rs

```diff
@@ -97,7 +95,11 @@ impl MasterEditionManager {
             )],
             Some(&self.authority.pubkey()),
             &[&self.authority],
-            context.last_blockhash,
+            context
+                .banks_client
+                .clone()
+                .get_new_latest_blockhash(&context.banks_client.get_latest_blockhash().await?)
```

Comment:
I'd wrap this in a function. It's repeated all over at this point :)

Reply:
Definitely a good idea!